Package/PackageSpecifier.cs

```diff
@@ -194,7 +198,7 @@ public override string ToString()
             if (this == VersionSpecifier.Any)
                 return "Any";
             if (this == VersionSpecifier.AnyRelease)
-                return "Any Release";
+                return "";
```

Comment:
Can we add "AnyRelease" / "AnyRelease" / "" to the VersionSpecifier.Parse method?

Reply:
Lets do this at a later point